setup.cfg

```diff
@@ -211,7 +211,130 @@ universal=1
 
 [tool:pytest]
 markers =
+    # special cases
     network: marks tests which require network connection
+    parallel: can be run in parallel
+    parallel_server_only: can be run in parallel (in server mode only)
+    skip_server: should not be run in server mode
+    # services
+    acm: acm service
```

Comment:
Can we get this feature without listing all services? This would be a pain to maintain, considering the list changes quite often